src/fontcache.cpp

```diff
@@ -243,6 +213,25 @@ static std::string GetDefaultTruetypeFontFile([[maybe_unused]] FontSize fs)
 #endif /* defined(WITH_FREETYPE) || defined(_WIN32) || defined(WITH_COCOA) */
 }
 
+static void TryLoadDefaultTrueTypeFont([[maybe_unused]] FontSize fs)
+{
+#if defined(WITH_FREETYPE) || defined(_WIN32) || defined(WITH_COCOA)
+	/* Find font file. */
+	std::string full_font = GetDefaultTruetypeFontFile(fs);
+	if (!full_font.empty()) {
+		FontCacheSubSetting* setting = GetFontCacheSubSetting(fs);
+
+		if (setting->size == 0) {
+			setting->size = FontCache::GetDefaultFontHeight(fs);
+		}
+
+		setting->font = full_font;
+		LoadFontHelper(fs);
+		setting->font = "";
+	}
+#endif /* defined(WITH_FREETYPE) || defined(_WIN32) || defined(WITH_COCOA) */
+}
+
```

Comment:
No need to add this, Load*Font() already handles the empty configuration case.

Reply:
I see now that you changed the underlying functions in that PR you did.